crates/shared/src/price_estimation/native/coingecko.rs

```diff
@@ -58,7 +58,8 @@ impl NativePriceEstimating for CoinGecko {
             let mut url = crate::url::join(&self.base_url, &self.chain);
             url.query_pairs_mut()
                 .append_pair("contract_addresses", &format!("{:#x}", token))
-                .append_pair("vs_currencies", "eth");
+                .append_pair("vs_currencies", "eth")
```

Comment:
I guess this we need to make configurable per network.

Reply:
Yes, I am going to address that in another upcoming PR :ok_hand: